_includes/community_lists.html

```diff
@@ -8,11 +8,11 @@
                             <img src="assets/github.svg" class="resource-logo" alt="github icon">
                         </div>
                         <div class="col-md-8 resource-text">
-                            <h3 class="resource-name">Jupyter Discourse</h3>
+                            <h3 class="resource-name">Jupyter GitHub</h3>
                             <p class="resource-desc">A place for the community to ask general questions about Jupyter software.</p>
```

Comment:
@Szauka  Good catch. I had my search and replace blinders on. 

We might want to update the description here Maybe "A place where the community collaborates on the development of Jupyter software". It's somewhat limiting because we also have the jupyterhub, jupyterlab, jupyter-incubator, etc. orgs on GitHub now.

Reply:
I (personally) would be dissapointed to not see a link to GitHub, when there is actually code in GitHub. 

Would be usefull to add more links to all the GitHub organisations?

Reply:
Sorry I wasn't clear. I'm fine with your change to point to GitHub. I'm only suggesting that the description text under the GitHub link change from what it currently says, "A place for the community to ask general questions about Jupyter software", to something like I wrote above. We're trying to encourage people to ask general questions on Discourse and use GitHub to collaborate on issues, code, project assets, etc.

Reply:
And I see you've pushed another commit, so, we're good!